Assets/Shopify/UIToolkit/Editor/ShopCredentialsVerifier.cs

```diff
@@ -0,0 +1,141 @@
+﻿namespace Shopify.UIToolkit.Editor {
+    using UnityEngine;
+    using UnityEditor;
+    using System;
+    using Shopify.Unity;
+    using Shopify.Unity.SDK;
+    using Shopify.Unity.SDK.Editor;
+
+    /// <summary>
+    /// A helper for the editor that allows verifying shop credentials
+    /// and drawing a UI to support the feature.
+    /// </summary>
+    public class ShopCredentialsVerifier {
+        private IShopCredentials _context;
```

Comment:
We've tried to follow the C# way of using Pascal Case for member variables. Can we change to `_Context`. Well actually maybe `_Credentials`.